webapp/app/__mocks__/axios.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  get: jest.fn(() => Promise.resolve({ data: [] })),
+  post: jest.fn(() => Promise.resolve({ data: {} })),
+  put: jest.fn(() => Promise.resolve({ data: {} })),
+  delete: jest.fn(() => Promise.resolve({ data: {} }))
+};
```

Comment:
I thinl it would be better to use moxios instead if using this magic mock.
If you prefer to keep this approach please use the already defined __mock__ folder on the top level, so we keep them on the same place.

Reply:
I`created the `__mock__` directory in the webapp top level in order to add the required mock implementations for third party libraries like axios, the implemented mock is the standard way to mock libraries with jest. Please clarify, thanks

Reply:
Please specify what Magic Mock means, not familiar with that expression, its an standard jest mock

Reply:
moved to top level folder